packages/cli/src/spec/DoctorSpec.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {loadOrCreateConfig} from '../lib/config';
+import * as mock from 'mock-fs';
+import {updateConfig} from '../lib/cmds/updateConfig';
+import {MockLog} from '@bubblewrap/core';
+import minimist = require('minimist');
+import {doctor} from '../lib/cmds/doctor';
+import {MockPrompt} from './mock/MockPrompt';
+import {enUS as messages} from '../lib/strings';
+
+describe('doctor', () => {
+  describe('#jdkDoctor', () => {
+    it('checks if a currect error is sent in case that the path given isn\'n valid', async () => {
```

Comment:
I'm fairly sure that "currect" is a typo, but I can't figure out what it's meant to be. Is it "correct"? In which case, what is a "correct error"?